.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+  - 2.3.1
+install:
+  - bundle install --without development test
```

Comment:
In the `Makefile` we have `bundle install --path .bundle --without development test`. I take it we don't need the `--path` arg here?

Reply:
That sounds good to me!